examples/export-data/main.ts

```diff
@@ -1,2 +1,183 @@
+import { EventEnvironment } from './utils/types';
+import { getProjectIDByName, createProject } from './actions/project'
+
+dotenv.config();
+const CLIENT_KEY = process.env.CLIENT_API_SECRET || '';
+const SERVER_KEY = process.env.SERVER_API_SECRET || '';
+const projectName = "Bits of Good Example Project";
+
+const main = async () => {
+    try {
+        await dbConnect();
+        console.log('Connected to MongoDB');
+
+        // Create the project if it doesn't exist
+        let projectId = await getProjectIDByName(projectName);
+        if (!projectId) {
+            console.log(`Project '${projectName}' does not exist. Creating new project`);
+            
+            const newProject: Partial<Project> = {
+                projectName: projectName,
+                clientApiKey: CLIENT_KEY,
+                serverApiKey: SERVER_KEY,
+                privateData: false,
+                deletionPolicy: -1,
+            };
+            
+            const createdProject = await createProject(newProject);
+            projectId = createdProject._id;
+            console.log(`Created new project with ID: ${projectId}`);
+        } else {
+            console.log(`Project ID found: ${projectId}`);
+        }
+
+        const { clickEvents, visitEvents, inputEvents, customEvents } = await fetchEvents();
+        
+        console.log(`Fetched ${clickEvents?.length} click events`)
+        console.log(`Fetched ${visitEvents?.length} visit events`)
+        console.log(`Fetched ${inputEvents?.length} input events`)
+        console.log(`Fetched ${customEvents?.length} custom events`)
+
+        for (const event of clickEvents || []) {
+            await ClickEventModel.create({
+                eventProperties: event.eventProperties,
+                category: event.category,
+                subcategory: event.subcategory,
+                projectId: projectId,
+                timestamp: event.createdAt ? new Date(event.createdAt) : new Date(),
+            });
+        }
+
+        for (const event of visitEvents || []) {
+            console.log(`event ${event}`)
+            await VisitEventModel.create({
+                eventProperties: event.eventProperties,
+                category: event.category,
+                subcategory: event.subcategory,
+                projectId: projectId,
+                timestamp: event.createdAt ? new Date(event.createdAt) : new Date(),
+            });
+        }
+
+        for (const event of inputEvents || []) {
+            await InputEventModel.create({
+                eventProperties: event.eventProperties,
+                category: event.category,
+                subcategory: event.subcategory,
+                projectId: projectId,
+                timestamp: event.createdAt ? new Date(event.createdAt) : new Date(),
+            });
+        }
+
+        for (const event of customEvents || []) {
+            await CustomEventModel.create({
+                eventTypeId: event.eventTypeId,
```

Comment:
If you are leaving the event type id in, then make sure you also export over the custom event types from bog analytics to the new database